locales/zh_CN.json

```diff
@@ -0,0 +1,1116 @@
+{
+  "lang": "zh_CN",
+  "rules": {
+    "accesskeys": {
+      "description": "确保每个 accesskey 属性值都是唯一的",
+      "help": "accesskey 属性值应当唯一"
+    },
+    "area-alt": {
+      "description": "确保图像映射中的 <area> 元素有替代文本",
+      "help": "<area> 元素必须有替代文本"
+    },
+    "aria-allowed-attr": {
+      "description": "确保元素的角色支持其 ARIA 属性",
+      "help": "元素仅能使用受支持的 ARIA 属性"
+    },
+    "aria-allowed-role": {
+      "description": "确保角色属性值符合所属元素",
+      "help": "ARIA 角色应该符合元素"
+    },
+    "aria-braille-equivalent": {
+      "description": "确保 aria-braillelabel 和 aria-brailleroledescription 有对应的非盲文的标签和描述",
+      "help": "aria-braille 相关属性必须有非盲文的对应属性"
+    },
+    "aria-command-name": {
+      "description": "确保每个 ARIA 按钮、链接和菜单项都有可访问名称",
+      "help": "ARIA 指令必须有一个可访问名称"
+    },
+    "aria-conditional-attr": {
+      "description": "确保 ARIA 属性的使用符合所属元素角色的特定规范",
+      "help": "ARIA 属性的使用必须符合元素的特定角色"
+    },
+    "aria-deprecated-role": {
+      "description": "确保页面元素不使用已弃用的角色属性",
+      "help": "不得使用已弃用的 ARIA 角色属性"
+    },
+    "aria-dialog-name": {
+      "description": "确保每个 ARIA 对话框和警示框节点都有可访问的名称",
+      "help": "ARIA 对话框和警示框节点应当有可访问的名称"
+    },
+    "aria-hidden-body": {
+      "description": "确保 <body> 不存在 aria-hidden=\"true\"",
+      "help": "<body> 上不得存在 aria-hidden=\"true\""
+    },
+    "aria-hidden-focus": {
+      "description": "确保设置为 ARIA 隐藏的元素不可聚焦，且不包含可聚焦的元素",
+      "help": "被设定为 ARIA 隐藏的元素不可聚焦，且不包含可聚焦的元素"
+    },
+    "aria-input-field-name": {
+      "description": "确保每个 ARIA 输入字段都有可访问的名称",
+      "help": "ARIA 输入字段必须有可访问的名称"
+    },
+    "aria-meter-name": {
+      "description": "确保每个 ARIA 计量器节点都有可访问的名称",
+      "help": "ARIA 计量器节点必须有可访问的名称"
+    },
+    "aria-progressbar-name": {
+      "description": "确保每个 ARIA 进度条节点都有可访问的名称",
+      "help": "ARIA 进度条节点必须有可访问的名称"
+    },
+    "aria-prohibited-attr": {
+      "description": "确保 ARIA 属性没有被禁止用于某元素的角色",
+      "help": "元素只能使用允许的 ARIA 属性"
+    },
+    "aria-required-attr": {
+      "description": "确保拥有 ARIA 角色的元素包含了所有必需的 ARIA 属性",
+      "help": "必须提供必需的 ARIA 属性"
+    },
+    "aria-required-children": {
+      "description": "确保需要子角色的 ARIA 角色元素包含它们",
+      "help": "某些 ARIA 角色必须包含特定的子元素"
+    },
+    "aria-required-parent": {
+      "description": "确保需要父角色的 ARIA 角色元素被它们包含",
+      "help": "某些 ARIA 角色必须被特定的父元素包含"
+    },
+    "aria-roledescription": {
+      "description": "确保 aria-roledescription 仅用于有隐式或显式角色的元素",
+      "help": "aria-roledescription 必须用于有语义角色的元素"
+    },
+    "aria-roles": {
+      "description": "确保所有包含 role 属性的元素都使用有效的值",
+      "help": "使用的 ARIA 角色必须符合有效的值"
```

Comment:
确保所有元素包含的角色属性值都是有效的
使用的 ARIA 角色必须是有效值

Reply:
I think it would be better to directly use the English word "role" here. Therefore, the role mentioned here refers to a specific attribute name within an element, such as the "id" attribute, which may be more difficult to understand when translated into Chinese.

Reply:
However, I know that this is a subjective opinion of mine. Personally, I think that when "role" is directly mentioned in the original text, I would translate it as "角色" in Chinese. However, if it is "role attribute" or any other "xxx attribute," I would retain the original English term.